frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js

```diff
@@ -456,6 +480,20 @@ class SchemaItem extends React.Component<{
                     if (this.props.create) return;
                     this.props.setFocus(this.props.focusIndex, 0);
                 }}
+                id={
+                    document.getElementById(
```

Comment:
It's probably worth refactoring this into a separate function, since we repeat it above in `componentDidMount`, and it's >10 lines of code.

Reply:
They are different actually. One is for setting id, the other is for getting id.

Reply:
Right, but you could refactor this into a method like `getId()` and then replace the code in the other place with something like this:

```
const node = document.getElementById(this.getId());`
```

Reply:
Refactored.